# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/Localization.webidl
# Commit: 92b87b0f776c
# Full Hash: 92b87b0f776c3049499038d380190351e2371324
# Author: Zibi Braniecki <zbraniecki@mozilla.com>
# Date: 2021-08-04 03:56:33
# Regressor Bug: 1613705
# File Overlap Count: 1
# Description:
#   Bug 1613705 - [localization] part2: Switch Localization class to use localization-ffi. r=emilio,nika
#   
#   Depends on D104788
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D104789
# ==============================================================================

diff -r 878b052ebeaa -r 92b87b0f776c dom/webidl/Localization.webidl
--- a/dom/webidl/Localization.webidl	Tue Aug 03 16:25:10 2021 +0000
+++ b/dom/webidl/Localization.webidl	Tue Aug 03 16:25:10 2021 +0000
@@ -92,16 +92,13 @@
    *                             default generators provided by Gecko.
    */
   [Throws]
-  constructor(sequence<DOMString> aResourceIds,
-              optional boolean aSync = false,
-              optional BundleGenerator aBundleGenerator = {});
+  constructor(sequence<UTF8String> aResourceIds,
+              optional boolean aSync = false);
 
   /**
    * A method for adding resources to the localization context.
-   *
-   * Returns a new count of resources used by the context.
    */
-  unsigned long addResourceIds(sequence<DOMString> aResourceIds);
+  void addResourceIds(sequence<DOMString> aResourceIds);
 
   /**
    * A method for removing resources from the localization context.